homeassistant/components/ezbeq/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up EzBEQ",
+        "description": "Set up your EzBEQ integration",
+        "data": {
+          "host": "ezbeq Host",
+          "port": "Port"
```

Comment:
Please use the common translation keys here and use `data_description` to add context